### PR TITLE
TC 07.3.11_2 Women/Tops/Pagination>list, grid mode

### PR DESCRIPTION
**Pull request by sinnadine**
https://trello.com/c/ZYnGO40z/622-tc-073112-women-tops-paginationlist-grid-mode

### CODE REVIEW

File: old_tests/womenTops.spec.js [hunk 1]
```diff
@@ -105,5 +105,12 @@ test.describe("womenTops", () => {
     await expect(page).toHaveURL(WOMEN_TOPS_URL)
     await expect(page).toHaveTitle('Tops - Women')
    }) 
-   
+   const { test, expect } = require('@playwright/test');
+
+    test('open page', async ({ page }) => {
```
Comment (S-Lanchik) [inline old_tests/womenTops.spec.js:110]: write the correct test name

File: old_tests/womenTops.spec.js [hunk 2]
```diff
@@ -105,5 +105,12 @@ test.describe("womenTops", () => {
...
+        await page.goto("https://magento.softwaretestingboard.com/women/tops-women.html");
```
Comment (S-Lanchik) [inline old_tests/womenTops.spec.js:111]: we start test from homePage

**Comment by S-Lanchik** [comment]
Also put PR link to trello card

**Comment by sinnadine** [comment]
[TC 07.3.11_2 Women/Tops/Pagination>list, grid mode](https://trello.com/c/ZYnGO40z/622-tc-073112-women-tops-paginationlist-grid-mode)

File: old_tests/womenTops.spec.js (hunk 1 above)
Comment (sinnadine) [reply old_tests/womenTops.spec.js]: resolved

**Review by S-Lanchik — APPROVED** [review]

**Pull request merged**